integreat_cms/cms/migrations/0060_user_distribute_sidebar_boxes.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.16 on 2023-01-30 13:07
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    """
+    Add setting to toggle the automatic distribution of sidebar boxes
+    """
+
+    dependencies = [
+        ("cms", "0059_add_poicategory_icon_color"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="user",
+            name="distribute_sidebar_boxes",
+            field=models.BooleanField(
+                default=False,
+                help_text="Enable this option to automatically distribute the sidebar boxes of content forms on screen resolutions wider than 2100px to make ideal use of the space on the screen.",
```

Comment:
Maybe this was a mistake of mine during the implementation, but actually this also works for Tablet sized screens, that have two sidebar columns beneath the editor. 
![grafik](https://user-images.githubusercontent.com/72705147/215595768-9591d823-d9ea-47a2-9c66-78cc86e508e9.png)
This goes for screen with a width between 768px (md) and 1600px (3xl)

Reply:
Good point! :+1:
I now adapted the wording.